kyo-combinators/shared/src/main/scala/kyo/Constructors.scala

```diff
@@ -191,7 +191,7 @@ extension (kyoObject: Kyo.type)
       * @return
       *   An effect that attempts to run the given effect and handles the Future to Async.
       */
-    def fromFuture[A: Flat, S](future: => Future[A] < S)(using Frame): A < (S & Async) =
+    def fromFuture[E <: Throwable: ClassTag, A: Flat, S](future: => Future[A] < S)(using Frame): A < (S & Async & Abort[E]) =
```

Comment:
I think this is a good example of a parameter that should not accept Kyos. By name can remain.